xml/art_sle_ha_install_quick.xml

```diff
@@ -399,9 +437,23 @@
     hardware. If no watchdog matches your hardware, or for testing purposes,
     use the software watchdog driver (<systemitem>softdog</systemitem>).
    </para>
+
+   <note>
+    <title>Softdog Not Supported</title>
+    <para>
+     &suse; does not not support <systemitem>softdog</systemitem> for
+     production. Use <systemitem>softdog</systemitem> only for testing
```

Comment:
&suse; does not not support [...] for production environments. Use  [...]  for testing purposes only. Before using the cluster in a production environment, replace the [...] module with your respective hardware module.